Sources/DataPipeline/CustomerIO+DataPipeline.swift

```diff
@@ -0,0 +1,227 @@
+import CioInternalCommon
+import Foundation
+
+public protocol DataPipelinePublicAPI: AutoMockable {
```

Comment:
The interface is doing almost the same thing as `CustomerIOInstance`, but it just move away from `Common` to `DataPipeline` because we no longer need those methods in `Common`.

Reply:
Removed for now